style.css

```diff
@@ -0,0 +1,47 @@
+body {
+	font-family: sans-serif;
+	align: center;
+}
+
+h1 {
+	background: white;
+	text-align: center;
+	padding: 15px;
+}
+
+p {
+	color: black;
+	margin-bottom: 10px;
+	font-size: 12px
+}
+
+p.main {
```

Comment:
@elysemclark Nice use of a class here. Classes (and id's) are the way we can target specific elements on our pages.